selfdrive/manager/manager.py

```diff
@@ -124,7 +136,7 @@ def manager_thread():
   if os.getenv("NOBOARD") is not None:
     ignore.append("pandad")
   if os.getenv("BLOCK") is not None:
-    ignore += os.getenv("BLOCK").split(",")
+    ignore += os.getenv("BLOCK","").split(",")
```

Comment:
there's already a check for this. either revert this or remove the if statement

Reply:
this is another type related oddity... since `os.getenv`'s return value is optional, it would potentially raise an exception in the event of it being None... i couldn't just remove the if statement though because it would add an empty string to the list which is unintended behavior. i'm going to submit a rewritten version of this section to solve all concerns